charts/redpanda/templates/tests/test-schemaregistry-status.yaml

```diff
@@ -49,5 +49,5 @@ spec:
         - "300"
         - --retry-max-time
         - "120"
-        - http://{{ include "redpanda.internal.domain" . }}:{{ .Values.listeners.schemaRegistry.port }}/subjects
+        - http://{{ include "redpanda.fullname" . }}:{{ .Values.listeners.schemaRegistry.port }}/subjects
```

Comment:
Why are we changing this test? Isn't connecting to the aggregate service preferred?

Reply:
This is just making this test match each of the other similar tests we have. If this is incorrect, then the other tests may also need to be updated similarly. I don't recall why this came up during this PR though... it definitely isn't related and could be handled in a separate PR.